tripal_chado/tests/src/Functional/api/ChadoOrganismAPITest.php

```diff
@@ -0,0 +1,182 @@
+<?php
+
+namespace Drupal\Tests\tripal_chado;
+
+use Drupal\Core\Url;
+use Drupal\Tests\BrowserTestBase;
+use Drupal\Core\Database\Database;
+use Drupal\tripal_chado\api\ChadoSchema;
+
+/**
+ * Testing the tripal_chado/api/tripal_chado.organism.api.inc functions.
+ *
+ * @group Tripal
+ * @group Tripal Chado
+ * @group Tripal API
+ */
+class ChadoOrganismAPITest extends BrowserTestBase {
+
+  protected $defaultTheme = 'stable';
+
+  /**
+   * Modules to enable.
+   * @var array
+   */
+  protected static $modules = ['tripal', 'tripal_chado'];
+
+  /**
+   * Schema to do testing out of.
+   * @var string
+   */
+  protected static $schemaName = 'testchado';
```

Comment:
We are phasing out using the `testchado` schema which you would have no way of knowing 🙈 I'll add in here the new way as suggestions (in two parts).

```suggestion
  protected $schemaName;
```

Reply:
Review suggestions appreciated and incorporated